rust/main/lander/src/adapter/chains/radix/precursor.rs

```diff
@@ -33,6 +33,41 @@ pub struct RadixTxPrecursor {
     pub tx_hash: Option<H512>,
 }
 
+impl std::fmt::Debug for RadixTxPrecursor {
```

Comment:
Implemented Debug manually to omit `encoded_arguments` so logs don't get huge.

Reply:
Try use `#[debug(skip)]` annotation on the field you would like to skip on logging.

Reply:
doesn't seem to work